### PR TITLE
[Squid][Automation][SMB]Load test

### PR DESCRIPTION
**Pull request by MohitBis**
Scenario: Setup Ceph cluster with SMB, run load test and check process id

### CODE REVIEW

**Comment by MohitBis** [comment]
Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-KXMRZU

File: tests/smb/smb_load.py [hunk 1]
```diff
@@ -318,15 +325,20 @@ def delete(smbclient, load_test_config, process):
...
+):
     """clean up
     Args:
         client (obj): client obj
         load_test_config (list): Smb load test parameter
     """
     try:
         load_dir = load_test_config["load_dir"]
-        cmd = f'smbclient -U user1%passwd //10.0.209.9/share1 -c "deltree {load_dir}"'
+        cmd = (
```
Comment (adityaramteke) [inline tests/smb/smb_load.py:338]: @MohitBis do you think converting smbclient to lib will be considering we are going to use it most of the test cases?

File: tests/smb/smb_load.py [hunk 2]
```diff
@@ -318,15 +325,20 @@ def delete(smbclient, load_test_config, process):
         raise OperationFailedError(f"delete operation failed, Error {e}")
 
 
-def cleanup(client, load_test_config):
+def cleanup(
```
Comment (pranavprakash20) [inline tests/smb/smb_load.py:328]: One question here Mohit, will this impact the existing cases? Are we using this somewhere else also? If its specific to this scenario then please ignore, rest looks good to me

Comment (MohitBis) [reply tests/smb/smb_load.py:328]: Yes @pranavprakash20 
It is specific to this test only.

File: suites/squid/smb/tier-2-smb-load.yaml [hunk 3]
```diff
@@ -0,0 +1,105 @@
+tests:
```
[the remark this answers is not in the document]
Comment (MohitBis) [reply suites/squid/smb/tier-2-smb-load.yaml:7]: Thanks @adityaramteke 
Updated

File: tests/smb/smb_load.py (hunk 1 above)
Comment (MohitBis) [reply tests/smb/smb_load.py:338]: Hi @adityaramteke 

We are using "deltree" command which is specific to load test only.

pysmb not comes with per defined function for deltree

**Review by pranavprakash20 — APPROVED** [review]
LGTM

File: tests/smb/smb_load.py (hunk 1 above)
Comment (adityaramteke) [reply tests/smb/smb_load.py:338]: Okay.

**Review by adityaramteke — APPROVED** [review]
LGTM

**Review by tintumathew10 — APPROVED** [review]

**Pull request merged**